static/app/actionCreators/pageFilters.tsx

```diff
@@ -53,39 +56,14 @@ type Options = {
 };
 
 /**
- * Represents the datetime portion of page filters staate
- */
-type DateTimeUpdate = {
-  start?: DateString;
-  end?: DateString;
-  statsPeriod?: string | null;
-  utc?: string | boolean | null;
-  /**
-   * @deprecated
-   */
-  period?: string | null;
```

Comment:
This becomes undeprecated actually. In fact, the only way to update date time is to pass period.